src/peer_connection.cpp

```diff
@@ -1255,7 +1267,8 @@ namespace libtorrent {
 			if (t && should_log(peer_log_alert::info))
 			{
 				peer_log(peer_log_alert::info, "ATTACH"
-					, "Delay loaded torrent: %s:", aux::to_hex(ih).c_str());
+					, "Delay loaded torrent: %s:"
+					, aux::to_hex(t->torrent_file().info_hash().get_best()).c_str());
```

Comment:
why isn't this just `ih.get_best()`?

Reply:
For hybrid torrents `ih` might only have the v1 hash while the torrent probably has both.